var/spack/repos/builtin/packages/py-torchdata/package.py

```diff
@@ -24,7 +24,7 @@ class PyTorchdata(PythonPackage):
     # https://github.com/pytorch/data#version-compatibility
     depends_on("python@3.7:3.10", type=("build", "run"))
     depends_on("py-torch@master", when="@main", type=("build", "run"))
-    depends_on("py-torch@1.12.1", when="@0.5.0", type=("build", "run"))  # ?
+    depends_on("py-torch@1.13.0", when="@0.5.0", type=("build", "run"))
```

Comment:
@tldahlgren I had a feeling we would see a new release of torch/torchvision to coincide with this torchdata release